terraform/test_runner/run_tests.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/google/fleetspeak/fleetspeak/src/e2etesting/setup"
+	"github.com/google/fleetspeak/fleetspeak/src/e2etesting/tests"
+	"os"
+	"time"
+)
+
+var (
+	numClients          = flag.Int("num_clients", 1, "Number of clients")
+	numServers          = flag.Int("num_servers", 1, "Number of servers")
+	masterServerAddress = flag.String("ms_address", "", "Address of master server")
+)
+
+func run() error {
+	serverHosts := make([]string, *numServers)
+	for i := 0; i < *numServers; i++ {
+		fmt.Scanf("%s", &serverHosts[i])
```

Comment:
This should not come from standard input (it's rather unconventional).

The servers should come from one of:
- From a flag (--servers=192.168.1.1,192.168.1.2)
- From a file (--servers_file=/tmp/servers.txt)